lib/avo/fields/concerns/frame_loading.rb

```diff
@@ -0,0 +1,14 @@
+module Avo
+  module Fields
+    module Concerns
+      module FrameLoading
+        extend ActiveSupport::Concern
+
+        def turbo_frame_loading
```

Comment:
Do you think this change will impact any other components @Paul-Bob?

Reply:
This code was extracted from `Avo::Fields::HasManyField::ShowComponent` to `Avo::Fields::Concerns::FrameLoading`. 

`Avo::Fields::HasManyField::ShowComponent` includes now `Avo::Fields::Concerns::FrameLoading` so any component that inherit from it and the component itself still have access to the code.

The extraction was made just to reuse the same code inside `Avo::Fields::HasOneField::ShowComponent`.

Any other unrelated components are not being impacted.

Reply:
@adrianthedev I misunderstood your question....

Yes, we should also apply this change on `app/components/avo/fields/has_many_field/show_component.html.erb`